addons/account/account_move_line.py

```diff
@@ -806,6 +807,11 @@ def prepare_move_lines_for_reconciliation_widget(self, cr, uid, lines, target_cu
                 amount = -amount
                 amount_currency = -amount_currency
 
+            # For already reconciled lines, don't use amount_residual(_currency)
+            if not line.account_id.reconcile:
```

Comment:
here, again, i don't get why you check line.account_id.reconcile instead of line.reconcile_id (as the comment say it should be)